### PR TITLE
[IOPAE-1445] ingestion services topics with eventhub

### PR DESCRIPTION
**Pull request by StefanoDoc**
services-topics ingestion using eventhub
#### List of Changes

New generic `findTopics` method in dao
New timerTrigger `serviceTopicsIngestor`
Add `modified_at` to topics db
#### How Has This Been Tested?
Local run 

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/io-services-cms-webapp/src/ingestion/service-topic-ingestor-handler.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      logPrefix,
+      "createServiceTopicIngestorHandler",
+    );
+    return await pipe(
+      serviceTopicDao.findAllTopics(),
+      TE.map(
+        flow(
+          O.fold(
+            () => writeTopicsOnEventHub([], producer, context),
+            (topics) => writeTopicsOnEventHub(topics, producer, context),
```
Comment (rodoviti) [inline apps/io-services-cms-webapp/src/ingestion/service-topic-ingestor-handler.ts:55]: nitpick (non-blocking):
```suggestion
          O.getOrElse<AllServiceTopics[]>(() => []),
          (topics) => writeTopicsOnEventHub(topics, producer, context),
```
PS: if you accept the suggested changes, you also need to import AllServiceTopics 😉

File: apps/io-services-cms-webapp/src/utils/ingestion/formatter/service-topic-avro-formatter.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    E.right,
+    E.chain((avroObj) =>
+      E.tryCatch(
+        () =>
+          avro.Type.forSchema(avroServiceTopics.schema as avro.Schema).toBuffer(
+            avroObj,
+          ),
+        E.toError,
+      ),
+    ),
```
Comment (rodoviti) [inline apps/io-services-cms-webapp/src/utils/ingestion/formatter/service-topic-avro-formatter.ts:39]: **nitpick (non-blocking):**
```suggestion
    E.tryCatchK(
      (avroObj) =>
        avro.Type.forSchema(avroServiceTopics.schema as avro.Schema).toBuffer(
          avroObj,
        ),
      E.toError,
    ),
```

Comment (StefanoDoc) [reply apps/io-services-cms-webapp/src/utils/ingestion/formatter/service-topic-avro-formatter.ts]: [b9f5014](https://github.com/pagopa/io-services-cms/pull/1061/commits/b9f5014288ec04666f9a87242e766966ef66e257)

File: apps/io-services-cms-webapp/src/ingestion/service-topic-ingestor-handler.ts (hunk 1 above)
Comment (StefanoDoc) [reply apps/io-services-cms-webapp/src/ingestion/service-topic-ingestor-handler.ts]: no longer needed after removing the option wrapper from dao

File: apps/io-services-cms-webapp/ServiceTopicsIngestor/function.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "bindings": [
+    {
+      "schedule": "0 * * * * *",
```
Comment (giamma1295) [inline apps/io-services-cms-webapp/ServiceTopicsIngestor/function.json:4]: is this cron expression right?

Comment (StefanoDoc) [reply apps/io-services-cms-webapp/ServiceTopicsIngestor/function.json]: [bd93794](https://github.com/pagopa/io-services-cms/pull/1061/commits/bd9379466a054007b05ef060a89ba3e5035181c0)

**Review by giamma1295 — APPROVED** [review]
LGTM

**Review by Krusty93 — APPROVED** [review]

**Pull request merged**